ansible/roles/test/tasks/telemetry.yml

```diff
@@ -0,0 +1,176 @@
+- debug: msg="String telemetry dial-in and dial-out tests..."
+
+
+- set_fact:
+    path_tmp_telemetry: /tmp/tmp_telemetry
+    path_ptf_run_dir: /tmp/test_telemetry
+    gitUrl_telemetry: https://github.com/Azure/sonic-telemetry.git
+    version_telemetry: aaa9188fda4b6f36dd5da00cdb240933fedae5ce
+    gitUrl_mgmt_framework: https://github.com/Azure/sonic-mgmt-framework.git
+    version_mgmt_framework: a4f1650d505c14d94872bd216f218e7c752de05c
+    go_fileName: go1.14.linux-amd64.tar.gz
+
+- name: Create tmp folder for telemetry
+  delegate_to: localhost
+  file: path="{{ path_tmp_telemetry }}" state=directory
+
+
+ #****************************************#
+ #           Prepareing                   #
+ #****************************************#
+- debug: msg="Prepareing the test code folder..."
+- block:
+  - name: pip pyangbind
+    delegate_to: localhost
+    become: true
+    pip:
+      name: pyangbind==0.6.0
+  - name: install pip3
+    delegate_to: localhost
+    become: true
+    apt:
+      name: python3-pip
+  - name: pip jinja2
+    delegate_to: localhost
+    become: true
+    pip:
+      name: jinja2>=2.10
+      executable: pip3
+
+  - name: Create sonic related project directory
+    delegate_to: localhost
+    file:
+      path: "{{ item }}"
+      state: directory
+    with_items:
+      - "{{ path_tmp_telemetry }}/sonic-telemetry"
+      - "{{ path_tmp_telemetry }}/sonic-mgmt-framework"
+
+  - name: Clone sonic related project
+    delegate_to: localhost
+    git:
+      repo: "{{ item.url }}"
+      dest: "{{ item.path }}"
+      version: "{{ item.ver }}"
+    with_items:
+      - { url: "{{ gitUrl_mgmt_framework }}", ver: "{{version_mgmt_framework}}", path: "{{ path_tmp_telemetry }}/sonic-mgmt-framework" }
+      - { url: "{{ gitUrl_telemetry }}", ver: "{{version_telemetry}}", path: "{{ path_tmp_telemetry }}/sonic-telemetry" }
+
+  - name: Download Go from https://dl.google.com/go/{{ go_fileName }}
+    delegate_to: localhost
+    get_url:
+      url: https://dl.google.com/go/{{ go_fileName }}
+      dest: "{{ path_tmp_telemetry }}/{{ go_fileName }}"
+  - name: Unarchive Go
+    delegate_to: localhost
+    become: true
+    unarchive:
+      src: "{{ path_tmp_telemetry }}/{{ go_fileName }}"
+      dest: /usr/local/
+      remote_src: yes

```

Comment:
the go binary should be built into the docker-ptf directly in the sonic-buildimage repo.

Reply:
The go binary here is in sonic-mgmt container to make (Makefile in sonic-telemetry) the dependency for running telemetry test